R/footer_dfe_auto.R

```diff
@@ -0,0 +1,100 @@
+#' Footer Function
+#'
+#' This function create a gov style footer for your page
+#' @param full Whenever you want to have blank footer or official gov version.
+#' Defaults to \code{FALSE}
+#' @return a footer html shiny object
+#' @keywords footer
+#' @export
+#' @examples
+#' if (interactive()) {
+#'   ui <- fluidPage(
+#'     shinyGovstyle::header(
+#'       main_text = "Example",
+#'       secondary_text = "User Examples",
+#'       logo = "shinyGovstyle/images/moj_logo.png"
+#'     ),
+#'     shinyGovstyle::banner(
+#'       inputId = "banner", type = "beta", "This is a new service"
+#'     ),
+#'     tags$br(),
+#'     tags$br(),
+#'     footer_dfe(full = TRUE)
+#'   )
+#'
+#'   server <- function(input, output, session) {
+#'     observeEvent(input$link_to_accessibility, {
+#'       updateTabsetPanel(session, "navlistPanel", selected = "Accessibility")
+#'     })
+#'   }
+#'
+#'   shinyApp(ui = ui, server = server)
+#' }
+footer_dfe <- function(full = FALSE) {
```

Comment:
For consistency with the `header`, I think let's stick to `footer` rather than `footer_dfe`.